lib/mix/tasks/gen.badges.ex

```diff
@@ -49,7 +49,7 @@ defmodule Mix.Tasks.Gen.Badges do
     path
     |> File.stream!()
     |> MyParser.parse_stream()
-    |> Stream.map(fn [name, description, begin_time, end_time, image_path, type] ->
+    |> Stream.map(fn [name, description, begin_time, end_time, image_path, type, tokens] ->
       {:ok, begin_datetime, _} = DateTime.from_iso8601("#{begin_time}T00:00:00Z")
```

Comment:
This change is already implemented in the master branch, so I would advise to first merge with that branch